src/components/ModalTitle/index.tsx

```diff
@@ -33,18 +34,30 @@ const StyledClose = styled(Close)`
   width: 35px;
 `
 
-const ModalTitle = ({
-  iconUrl,
-  title,
-  onClose,
-}: {
+const GoBackWrapper = styled.div`
+  margin-right: 15px;
+
+  :hover {
+    cursor: pointer;
+  }
+`
+
+type Props = {
   title: string
-  iconUrl: string
+  goBack?: () => void
+  iconUrl?: string
   onClose?: () => void
-}): React.ReactElement => {
+}
+
+const ModalTitle = ({ goBack, iconUrl, title, onClose }: Props): React.ReactElement => {
   return (
     <StyledRow align="center" grow>
       <TitleWrapper>
+        {goBack && (
+          <GoBackWrapper onClick={goBack}>
+            <Icon type="arrowLeft" size="md" />
+          </GoBackWrapper>
```

Comment:
why not `<IconButton>` as it's used by the close button?

Reply:
I can not remove GoBackWrapper, but I can make use of IconButton to be consistent.